src/widget/wtrackmenu.cpp

```diff
@@ -1257,44 +1257,42 @@ bool WTrackMenu::featureIsEnabled(Feature flag) const {
 
     switch (flag) {
     case Feature::AutoDJ:
-        return m_pTrackModel->hasCapabilities(TrackModel::TRACKMODELCAPS_ADDTOAUTODJ);
+        return m_pTrackModel->hasCapabilities(TrackModel::Capability::AddToAutoDJ);
     case Feature::LoadTo:
         return m_pTrackModel->hasCapabilities(
-                       TrackModel::TRACKMODELCAPS_LOADTODECK) ||
+                       TrackModel::Capability::LoadToDeck) ||
                 m_pTrackModel->hasCapabilities(
-                        TrackModel::TRACKMODELCAPS_LOADTOSAMPLER) ||
+                        TrackModel::Capability::LoadToSampler) ||
                 m_pTrackModel->hasCapabilities(
-                        TrackModel::TRACKMODELCAPS_LOADTOPREVIEWDECK);
+                        TrackModel::Capability::LoadToPreviewDeck);
     case Feature::Playlist:
-        return m_pTrackModel->hasCapabilities(
-                TrackModel::TRACKMODELCAPS_ADDTOPLAYLIST);
     case Feature::Crate:
         return m_pTrackModel->hasCapabilities(
-                TrackModel::TRACKMODELCAPS_ADDTOCRATE);
+                TrackModel::Capability::AddToTrackSet);
     case Feature::Remove:
         return m_pTrackModel->hasCapabilities(
-                       TrackModel::TRACKMODELCAPS_REMOVE) ||
+                       TrackModel::Capability::Remove) ||
                 m_pTrackModel->hasCapabilities(
-                        TrackModel::TRACKMODELCAPS_REMOVE_PLAYLIST) ||
+                        TrackModel::Capability::RemovePlaylist) ||
                 m_pTrackModel->hasCapabilities(
-                        TrackModel::TRACKMODELCAPS_REMOVE_CRATE);
+                        TrackModel::Capability::RemoveCrate);
     case Feature::Metadata:
-        return m_pTrackModel->hasCapabilities(TrackModel::TRACKMODELCAPS_EDITMETADATA);
+        return m_pTrackModel->hasCapabilities(TrackModel::Capability::EditMetadata);
     case Feature::Reset:
-        return m_pTrackModel->hasCapabilities(TrackModel::TRACKMODELCAPS_EDITMETADATA) &&
-                m_pTrackModel->hasCapabilities(TrackModel::TRACKMODELCAPS_RESETPLAYED);
+        return m_pTrackModel->hasCapabilities(TrackModel::Capability::EditMetadata) &&
+                m_pTrackModel->hasCapabilities(TrackModel::Capability::ResetPlayed);
```

Comment:
I think you can use a bitwise OR and combine this into a single `hasCapabilities` call.